client/config/parameters.yml

```diff
@@ -0,0 +1,69 @@
+# This file is auto-generated during the composer install
```

Comment:
I think this would normally be apart of our .gitignore

Probably shouldn't commit this to be safe.